content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+<html>
+  <head>
+    <title>Easily build and deploy your Sphinx docs</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="landing/sphinx" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Learn more about Sphinx & Read the Docs hosting" />
+  </head>
+  <body>
+
+{#
+  View this page at http://localhost:8080/landing/sphinx/index.html
+#}
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+
+<section>
+  {% call basic.header_image(
+    header="Quickly deploy your Sphinx docs across multiple versions",
+    image="https://cdn.dribbble.com/users/876899/screenshots/2141696/readthedocs-thumb.png",
+    image_alt="Marketing introduction header",
+    vertical_align="middle")
+  %}
+    <p>
+        Beautiful docs for your team are just a <code>git push</code> away.
+    </p>
+
+    <p class="ui basic padded vertical segment">
+      <a href="#" class="ui large stackable primary button">
+        Register an account
+      </a>
+      <a href="https://docs.readthedocs.io/en/stable/tutorial/index.html" class="ui large stackable button">
+        Start our tutorial
+      </a>
+    </p>
+  {% endcall %}
+</section>
+
+<section id="features">
+  {% call marketing.highlight_list() %}
+    {% call marketing.highlight_item(
+        icon="olive fa-cauldron",
+        icon_style="--fa-secondary-color: violet; --fa-secondary-opacity: 1;",
+        header="Preview each pull request",
+        color="primary") %}
+    {% markdown %}
+    Every time you create a new Pull Request on GitHub or GitLab,
```

Comment:
```suggestion
    Every time you create a new pull request on GitHub or GitLab,
```

if we want to emphasize "pull request" we can use `<em>` or something similar.